sync/sync.py

```diff
@@ -253,6 +227,9 @@ def create_resource(dest_prefix, file, versions):
         resource = resource_template.render(component_versions_json=serialize)
     elif ".md" in file:
```

Comment:
I forgot to do fix this as well. to use `endswith` instead of `in`

Reply:
Thanks @afrittoli 
/lgtm

Reply:
Nice catch, thanks. Fixed.